extensions/cdc-debezium/src/test/java/com/hazelcast/jet/cdc/AbstractCdcIntegrationTest.java

```diff
@@ -144,6 +148,35 @@ protected static int fixPortBinding(GenericContainer<?> container, Integer defau
         return port;
     }
 
+    /**
+     * Start the container after it has been stop
+     *
+     * It's necessary to use this method after calling {@link #stopContainer(GenericContainer)} with
+     * {@link #fixPortBinding(GenericContainer, Integer)}. If fixed port is not used you can use
+     * {@link GenericContainer#start()}
+     *
+     * @param container container to start
+     */
+    protected static void startContainer(GenericContainer<?> container) {
```

Comment:
This is only needed when the container is restarted. In that case if we query the container for the  mapped port we get a different port after the restart of the container.

What some of these tests test is a restart of the database while a job is running - it is configured to a fixed port and it can't be updated after the job is submitted.

Nevertheless, I have realized that there is still an issue when multiple builds run at the same time on a single machine - when the container is stopped it releases the port. other container started from another module can claim it and if it doesn't finish within the retries then the test might fail.

Maybe instead we could make use of `container.getDockerClient().restartContainerCmd(container.getContainerId());`. 

I will look into this more.